# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scroll-to-text-fragment/scroll-to-text-fragment.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/scroll-to-text-fragment/scroll-to-text-fragment.html.ini
--- a/testing/web-platform/meta/scroll-to-text-fragment/scroll-to-text-fragment.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/scroll-to-text-fragment/scroll-to-text-fragment.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,5 +1,7 @@
 [scroll-to-text-fragment.html]
-  expected: [TIMEOUT, OK]
+  expected:
+    if os == "mac": ["OK", "TIMEOUT"]
+    [TIMEOUT, OK]
   [Test navigation with fragment: Fragment directive with percent encoded syntactical characters "&,-" should match text.]
     expected: [FAIL, TIMEOUT, NOTRUN]
 
@@ -117,7 +119,6 @@
   [Test navigation with fragment: Non-matching text directive with existing element fragment should scroll to element.]
     expected:
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Generic fragment directive with existing element fragment should scroll to element.]
@@ -129,7 +130,6 @@
   [Test navigation with fragment: Multiple match text directive disambiguated by suffix should match the suffixed text.]
     expected:
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Exact text with suffix should match text.]
@@ -137,14 +137,14 @@
 
   [Test navigation with fragment: Text directive should match when context terms are separated by node boundaries.]
     expected:
-      if os == "mac": ["FAIL", "NOTRUN", "TIMEOUT"]
-      if os == "linux": ["FAIL", "TIMEOUT", "NOTRUN"]
-      [NOTRUN, FAIL, TIMEOUT]
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
+      if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
+      if os == "win": ["TIMEOUT", "FAIL", "NOTRUN"]
+      [FAIL, NOTRUN, TIMEOUT]
 
   [Test navigation with fragment: Non-matching text directive with nonexistent element fragment should not match and not scroll.]
     expected:
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Non-whole-word exact text with spaces should not match.]
@@ -155,13 +155,14 @@
 
   [Test navigation with fragment: Text directive should not scroll to hidden text.]
     expected:
-      if os == "linux": ["FAIL", "NOTRUN", "TIMEOUT"]
-      [NOTRUN, FAIL, TIMEOUT]
+      if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
+      if os == "win": ["NOTRUN", "FAIL", "TIMEOUT"]
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
+      [FAIL, NOTRUN, TIMEOUT]
 
   [Test navigation with fragment: Multiple match text directive disambiguated by prefix should match the prefixed text.]
     expected:
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Text range with no context should match text.]
@@ -182,8 +183,8 @@
 
   [Test navigation with fragment: Text directive should match text within shadow DOM.]
     expected:
-      if os == "mac": ["TIMEOUT", "NOTRUN", "FAIL"]
-      if os == "linux": ["FAIL", "TIMEOUT", "NOTRUN"]
+      if os == "linux": ["FAIL", "NOTRUN", "TIMEOUT"]
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
       [NOTRUN, FAIL, TIMEOUT]
 
   [Test navigation with fragment: Text range with non-matching startText should not match.]
@@ -199,9 +200,8 @@
 
   [Test navigation with fragment: Multiple match text directive disambiguated by prefix and suffix should match the text with the given context.]
     expected:
-      if os == "mac": ["FAIL", "NOTRUN", "TIMEOUT"]
+      if os == "linux": ["FAIL", "NOTRUN", "TIMEOUT"]
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["TIMEOUT", "NOTRUN", "FAIL"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Text range with non-matching endText should not match.]
@@ -226,7 +226,7 @@
 
   [Test navigation with fragment: Fragment directive with all TextMatchChars should match text.]
     expected:
-      if os == "android": ["TIMEOUT", "NOTRUN", "FAIL"]
+      if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Exact text with percent encoded spaces should match text.]
@@ -239,7 +239,8 @@
 
   [Test navigation with fragment: Text directive should not scroll to display none text.]
     expected:
-      if os == "linux": ["TIMEOUT", "NOTRUN", "FAIL"]
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
+      if os == "linux": ["FAIL", "NOTRUN", "TIMEOUT"]
       [NOTRUN, FAIL, TIMEOUT]
 
   [Test navigation with fragment: Uppercase TEXT directive should not parse as a text directive.]
@@ -252,14 +253,17 @@
 
   [Test navigation with fragment: Fragment directive with percent encoded non-ASCII unicode character should match text.]
     expected:
-      if os == "android": ["FAIL", "NOTRUN", "TIMEOUT"]
+      if os == "android": ["TIMEOUT", "FAIL", "NOTRUN"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Exact text with no context should match text.]
     expected: [FAIL, TIMEOUT, NOTRUN]
 
   [Test navigation with fragment: Text directive should horizontally scroll into view.]
-    expected: [NOTRUN, FAIL, TIMEOUT]
+    expected:
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
+      if os == "linux": ["TIMEOUT", "NOTRUN", "FAIL"]
+      [NOTRUN, FAIL, TIMEOUT]
 
   [Test navigation with fragment: Text directive followed by non-text directive should match text.]
     expected: